lib/configproxy.js

```diff
@@ -623,12 +613,29 @@ class ConfigurableProxy extends EventEmitter {
     }
   }
 
+  handleMetrics(req, res) {
+    if (req.url === "/metrics") {
+      return this.metrics.render().then((s) => {
+        res.writeHead(200, { "Content-Type": "text/plain; version=0.0.4" });
```

Comment:
What is version 0.0.4 referring to?

Reply:
That's the Prometheus text version. I found prom-client has that string. I'm replacing it now.
https://github.com/siimon/prom-client/blob/96f7495d66b1a21755f745b1367d3e530668a957/lib/registry.js#L146-L148